homeassistant/components/tuneblade_remote/coordinator.py

```diff
@@ -0,0 +1,54 @@
+"""Integration to integrate TuneBlade Remote devices with Home Assistant."""
+
+from datetime import timedelta
+import logging
+
+from aiohttp import ClientError
+from pytuneblade import TuneBladeApiClient
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+class TuneBladeDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class TuneBladeDataUpdateCoordinator(DataUpdateCoordinator[..]):
```
Can we type the return type of `_async_update_data`?

Reply:
Both? That's done